packages/contracts/test/plugins/governance/admin/admin.ts

```diff
@@ -96,20 +96,54 @@ describe('Admin plugin', function () {
   });
 
   describe('plugin interface: ', async () => {
-    it('supports plugin cloneable interface', async () => {
+    it('does not support the empty interface', async () => {
+      expect(await plugin.supportsInterface('0x00000000')).to.be.eq(false);
+    });
+
+    it('supports the `IERC165Upgradeable` interface', async () => {
       // @ts-ignore
-      const IPlugin = await hre.artifacts.readArtifact('IPlugin');
+      const IERC165Upgradeable = await hre.artifacts.readArtifact(
+        'IERC165Upgradeable'
+      );
+      const iface = new ethers.utils.Interface(IERC165Upgradeable.abi);
 
+      expect(await plugin.supportsInterface(getInterfaceID(iface))).to.be.eq(
+        true
+      );
+    });
+
+    it('supports the `IPlugin` interface', async () => {
+      // @ts-ignore
+      const IPlugin = await hre.artifacts.readArtifact('IPlugin');
       const iface = new ethers.utils.Interface(IPlugin.abi);
 
       expect(await plugin.supportsInterface(getInterfaceID(iface))).to.be.eq(
         true
       );
     });
 
-    it('supports admin address plugin interface', async () => {
+    it('supports the `IProposal` interface', async () => {
+      // @ts-ignore
+      const IProposal = await hre.artifacts.readArtifact('IProposal');
+      const iface = new ethers.utils.Interface(IProposal.abi);
+
+      expect(await plugin.supportsInterface(getInterfaceID(iface))).to.be.eq(
```

Comment:
didn't we say that when you want to check `interfaceID`, you would need to make 3 calls (one for ERC165, one for the `interfaceId`, one for `0x00000000` where the first 2 should return true, the last one false. Without such checks, we really are not testing really well ! 

I will review the test cases once this is rewritten.

Reply:
We already do. This is part of every `describe('plugin interface`:
```
it('does not support the empty interface', async () => {
      expect(await plugin.supportsInterface('0x00000000')).to.be.eq(false);
    });
```